lib/css/components/sidebar-widgets.less

```diff
@@ -0,0 +1,245 @@
+.s-sidebarwidget {
+    --_sw-content-px: calc(var(--su16) - var(--su1)); // subtract 1px for border
+    --_sw-content-spacing-inner: var(--su12); // the spacing between two adjacent simple items
+    --_sw-content-spacing-outer: var(--su16); // the spacing at the start/end of a group of simple items, as well as between a complex item and its separator line
+    --_sw-bc: var(--bc-medium);
+    --_sw-content-bc: var(--bc-light);
```

Comment:
IMO, a little redundancy is a good thing here. For instance, I see that `--_sw-content-bc` is being used for both the `content` and `header` child elements. I'd prefer to do something like:

```
--_sw-content-bc: var(--bc-light);
--_sw-header-bc: var(--_sw-content-bc);
```

This would be so a contributor can quickly parse that a) the same value us used for the border color in both `content` and `header` and b) modifying `--_sw-content-bc` will also modify the value for `--_sw-header-bc`.